numpy/typing/_platform_aliases.py

```diff
@@ -0,0 +1,62 @@
+"""
+A module with system-specific type aliases.
+
+Notes
+-----
+A dynamically generated stub file (with actual values) is created
+upon running setup.py.
```

Comment:
If this file is replaced by the generated one at installation, what is the point in this version?

Reply:
Its purpose is to provide runtime-accessible values so we can just export them to `numpy/typing/__init__.pyi`.
This way we don't have to worry about whether `typing.TYPE_CHECKING` is `True` or `False`.

Reply:
I'd also like to note that the script run during the numpy installation generates 
`_platform_aliases.pyi`, not `_platform_aliases.py`.

Reply:
It would be great if you could draw attention to the fact that the generated file has a different name in a comment.